Resources/views/Collector/swiftmailer.html.twig

```diff
@@ -1,77 +1,175 @@
+        <style>
+            h3 { margin-top: 2em; }
+            h3 span { color: #999; font-weight: normal; }
+            h3 small { color: #999; }
+            h4 { font-size: 14px; font-weight: bold; }
+            .card { background: #F5F5F5; margin: .5em 0 1em; padding: .5em; }
+            .card .label { display: block; font-size: 13px; font-weight: bold; margin-bottom: .5em; }
+            .card .card-block { margin-bottom: 1em; }
+        </style>
+    {% endif %}
+
+    <h2>E-mails</h2>
+
+    {% if not collector.mailers %}
+        <div class="empty">
+            <p>No e-mail messages were sent.</p>
+        </div>
+    {% endif %}
+
+    {% if profiler_markup_version == 1 %}
+        <table>
+            <thead>
                 <tr>
-                    <th>{{ name }}{{ collector.isDefaultMailer(name) ? ' (default mailer)' : '' }}</th>
-                    <td>{{ collector.messageCount(name) }} {{ collector.isSpool(name) ? 'spooled' : 'sent' }}</td>
+                    <th scope="col">Mailer</th>
+                    <th scope="col">Messages</th>
                 </tr>
-            {% endfor %}
-        </tbody>
-    </table>
+            </thead>
+            <tbody>
+                {% for name in collector.mailers %}
+                    <tr>
+                        <th>{{ name }}{{ collector.isDefaultMailer(name) ? ' (default mailer)' : '' }}</th>
+                        <td>{{ collector.messageCount(name) }} {{ collector.isSpool(name) ? 'spooled' : 'sent' }}</td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    {% else %}
+        {% if collector.mailers|length > 1 %}
+            <h3>Messages sent by each mailer</h3>
+
+            <div class="metrics">
+                {% for name in collector.mailers %}
+                    <div class="metric">
+                        <span class="value">{{ collector.messageCount(name) }}
+                        {% if collector.messageCount(name) > 0 %}
+                            <span class="unit">{{ collector.isSpool(name) ? 'spooled' : 'sent' }}</span>
+                        {% endif %}
+                        </span>
+                        <span class="label">
+                            {{ name }} {{ collector.isDefaultMailer(name) ? ' (default app mailer)' : '' }}
+                        </span>
+                    </div>
+                {% endfor %}
+            </div>
+        {% else %}
+            <div class="metrics">
+                {% for name in collector.mailers %}
+                    <div class="metric">
+                        <span class="value">{{ collector.messageCount(name) }}
+                        {% if collector.messageCount(name) > 0 %}
+                            <span class="unit">{{ collector.isSpool(name) ? 'spooled' : 'sent' }}</span>
+                        {% endif %}
+                        </span>
+                        <span class="label">sent messages</span>
```

Comment:
> sent messages would look weird here. The block will say 5 spooled sent messages or 5 sent sent messages

I agree with you. In https://github.com/javiereguiluz/SwiftmailerBundle/commit/7b5178decb95c0256e9c2ddbf14306c9b491bd30 I've redesigned this section a bit.

**Regular use case**: 1 mailer

Now we just say: `N spooled messages` or `N sent messages`

![profiler-single-mailer](https://cloud.githubusercontent.com/assets/73419/10045682/fdbf2642-6204-11e5-8c1b-c01ef62efb2b.png)

**Multiple-mailer user case**:

We display a table because we want to show a lot of details (mailer name, num. of messages, if they were sent or spooled, etc.)

![profiler-multiple-mailers](https://cloud.githubusercontent.com/assets/73419/10045697/0ba22e80-6205-11e5-9f08-0317940ef42d.png)

![profiler-multiple-mailers-messages](https://cloud.githubusercontent.com/assets/73419/10045703/118ac7b2-6205-11e5-811e-3db397795983.png)